### DISCUSSION TITLE
Possible for Clean_Transmission (or torrent cleanup) to Ignore File?

### DISCUSSION THREAD

**Discussion by rebeltaz** [Flexget/Flexget#3852] [Q&A] [answered]
I am running Flexget v3.1.134 and still using the Clean_Transmission plugin. (I didn't know it was deprecated, but...) Is it possible to tell that plugin to NOT delete a certain file? I would like transmission to continue seeding Fanimatrix indefinitely, but continue to remove all the other torrents like usual. Right now I have set that one file to read-only, but I don't know if that's going to work or cause other issues. Any help or tips will be greatly appreciated. Thanks.

**Comment by gazpachoking** [comment, marked as answer]
There isn't a way with clean_transmission, that's the reason we switched from it. If you switch to the new way you'll be able to filter however you want though. https://flexget.com/UpgradeActions#clean_transmission

**Reply by rebeltaz** [reply]
Yeah. That's what I figured. I had taken a look at the new TorrentCleanup page before posting here, but I didn't see anything there about this specific action. Would you know how or where I could find that using the new plugin?

**Reply by gazpachoking** [reply]
You can use any of the normal filter plugins to define what should be cleaned up. You can start with the basic rules on the wiki page, then add e.g. regexp plugin to reject anything that has fanimatrix in the title.
